marimbabot_audio/launch/marimbabot.launch

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-    <node name="audio_from_lilypond" pkg="marimbabot_audio" type="audio_from_lilypond.py"/>
-    <node name="audio_audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
-        <param name="device" value="" />
-        <param name="format" value="wave" />
-        <param name="channels" value="1" />
-        <param name="depth" value="16" />
-        <param name="sample_rate" value="44100" />
-    </node>
+
+  <!-- list the devices: arecord -l -->
+    <group ns="audio_node">
+      <node name="note_audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
+          <param name="device" value="plughw:3,0" />
```

Comment:
This seems very device specific

Reply:
yes, it seems the only way to specify the hardware currently, and the sequence to plugin the used device could change the id of the device, it is indeed uncomfortable.